stdlib/tkinter/__init__.pyi

```diff
@@ -3331,9 +3331,32 @@ class PhotoImage(Image, _PhotoImageLike):
     def blank(self) -> None: ...
     def cget(self, option: str) -> str: ...
     def __getitem__(self, key: str) -> str: ...  # always string: image['height'] can be '0'
-    def copy(self) -> PhotoImage: ...
-    def zoom(self, x: int, y: int | Literal[""] = "") -> PhotoImage: ...
-    def subsample(self, x: int, y: int | Literal[""] = "") -> PhotoImage: ...
+    if sys.version_info >= (3, 13):
+        def copy(
+            self,
+            *,
+            from_coords: Iterable[int] | None = None,
+            zoom: int | tuple[int, int] | None = None,
+            subsample: int | tuple[int, int] | None = None,
+        ) -> PhotoImage: ...
+        def subsample(self, x: int, y: Literal[""] = "", *, from_coords: Iterable[int] | None = None) -> PhotoImage: ...
+        def zoom(self, x: int, y: Literal[""] = "", *, from_coords: Iterable[int] | None = None) -> PhotoImage: ...
+        def copy_replace(
+            self,
+            sourceImage: PhotoImage,
```

Comment:
The image doesn't need to be a `PhotoImage` object for this method to work, so it should be:

```suggestion
            sourceImage: _ImageSpec,
```

For example, on my linux system, this shows a light bulb image:

```
>>> import tkinter
>>> root = tkinter.Tk()
>>> img = tkinter.PhotoImage(width=50, height=50)
>>> label = tkinter.Label(root, image=img)
>>> label.pack()
>>> tkinter.image_names()
('pyimage1', '::tk::icons::information', '::tk::icons::error', '::tk::icons::warning', '::tk::icons::question')
>>> img.copy_replace('::tk::icons::information')
```

Reply:
The docstring said ` """Copy a region from the source image (which must be a PhotoImage)`, so that's why I put this, good catch!

Reply:
Actually, perhaps we want `PhotoImage | str`, thoughts?